### PR TITLE
#1094 privacy policy in footer

### PR DESCRIPTION
**Pull request by maryanchuk04**
dev
## GitHub Project

* [Main GitHub Project ticket](https://github.com/EventsExpress/EventsExpress/projects/2)

## Code reviewers

- [ ] @sand0 
- [ ] @iuriikhrystiuk 
### Second Level Review

- [ ] @Yrja 

## Summary of issue
Create Privacy page

## Summary of change
Create privacy folder and privacy.js/css. 
Add new Route in app.js

## Testing approach

ToDo

## CHECK LIST
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: EventsExpress/appsettings.json [hunk 1]
```diff
@@ -2,13 +2,32 @@
   "AppBaseUrl": {
```
Comment (iuriikhrystiuk) [inline EventsExpress/appsettings.json:2]: Undo this file. We do not keep secrets in source control.

File: EventsExpress/ClientApp/src/components/privacy/privacy.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        We reserve the right to update or change our Privacy Policy at any time and you should check this Privacy Policy periodically. Your continued use of the Service after we post any modifications to the Privacy Policy on this page will constitute your acknowledgment of the modifications and your consent to abide and be bound by the modified Privacy Policy. If you do not consent to such changes you should not continue to use the services.
+        </p>
+        <h4>Enforcement</h4>
+        <p>We regularly review our own compliance with our Privacy Policy. If you submit a formal written complaint with your contact information, we will try our best to resolve the issue.</p>
+        <h4>Contact Us</h4>
+        <p>If you have any questions about this Privacy Policy or notice activity against it, please contact us. 
+            
+        </p>
+
+        <p>address</p>
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/components/privacy/privacy.js]: Should there be some real address? If not - just remove it, or add e-mail of some sorts.

File: EventsExpress/ClientApp/src/components/privacy/privacy.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
+.privacy{
+   margin:20px 0; 
+   font-size: 18px;
+}
+.privacy p{
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/components/privacy/privacy.css]: Combine multiple similar styles to a sinlge selector or class please. And format the code if possible - alt+shift+h in vs code.

File: EventsExpress/ClientApp/src/components/app/app.js [hunk 4]
```diff
@@ -94,7 +95,9 @@ class App extends Component {
                                 <Route path='/drafts' component={this.UserRoleSecurity(EventDraftListWrapper)} />
                                 <Route path='/unauthorized' component={Unauthorized} />
                                 <Route path='/forbidden' component={Forbidden} />
+                                <Route path='/privacy' component = {privacy}/>
```
Comment (iuriikhrystiuk) [inline EventsExpress/ClientApp/src/components/app/app.js:98]: You only added a route. Where is it used?

**Review by iuriikhrystiuk — APPROVED** [review]

**Comment by AndrewTheM** [comment]
Trying to reopen

**Pull request merged**